### ISSUE TITLE
Right Click Harvest Not Working

### ISSUE THREAD

**Issue by JC1OZ** [MatrexsVigil/phc2crops#55]
I'm using the mod with
![image](https://user-images.githubusercontent.com/83477320/137819209-e9a1eb92-dc68-4ac6-b6d4-496d0c6142f4.png)
![image](https://user-images.githubusercontent.com/83477320/137819255-e548e4ef-74be-4b75-89a8-b92acd543951.png)
+abnormals core any reasons?

**Comment by JC1OZ** [comment]
I checked config i should still be able to harvest everything is on default

**Comment by trollkind** [comment]
Yeah, I was disappointed when the Kama of Tinkers Construct didn't work with PHC2 crops.
Looks like other crop mods have the same problem, here's a crosspost in Mystical Agriculture where Knightminer also explains how the Kama works for replanting. I hope this can be fixed at some point from either side.
https://github.com/BlakeBr0/MysticalAgriculture/issues/569
